### ISSUE TITLE
Add markup highlight style to docs? 

### ISSUE THREAD

**Issue by vxnick** [tomfran/typo#17]
I noticed that with a default configuration, syntax highlighting on the light/default theme can result in unreadable text (white text on a very light background).

I wasn't sure how best to fix this in CSS so I ended up looking at your website's repo and noticed you set a markup highlight style that also fixes this for me, so is this worth mentioning in the documentation if it can't be fixed with CSS directly?

**Comment by tomfran** [comment]
You mean using `algol` theme for syntax highlighting? 

Strange, I don't see any problems setting it to my local website:

<img width="300" alt="Screenshot 2024-09-02 at 22 47 43" src="https://github.com/user-attachments/assets/e391703b-f885-4941-bea2-4fbf26eca89f">

<img width="300" alt="Screenshot 2024-09-02 at 22 47 38" src="https://github.com/user-attachments/assets/8f157ee4-745a-410d-af15-4cea757c9231">


Perhaps I am missing something.

**Comment by vxnick** [comment]
Based on what I found, it looks like the highlighter's [default style](https://gohugo.io/functions/transform/highlight/#options) is `monokai`. I noticed you also [override your style](https://github.com/tomfran/blog/blob/main/hugo.toml#L82) to `bw` so wondered if you had encountered the same issue originally.

To reproduce, either change `markup.highlight.style = monokai` or remove it from `hugo.toml`. This is what I saw until I set `markup.highlight.style` to something else.

![Screenshot 2024-09-03 at 19 22 56](https://github.com/user-attachments/assets/5ae2edce-7dc9-467d-8f6a-eff702ab6411)

I see this on code blocks using `bash` and `python` formatting and also your own code block if I annotate it as Rust with `<three-backticks>rust`

![Screenshot 2024-09-03 at 19 26 02](https://github.com/user-attachments/assets/416d1b78-792b-4df6-adf5-985ba1fc9e3e)

This works on plain backquoted code blocks without a language annotation.

Thanks!
